website/static/js/projectSettings.js

```diff
@@ -59,6 +73,17 @@ var NodeCategorySettings = oop.extend(
                 err: error
             });
         },
+        updateTitleError: function() {
+            document.getElementById('title-input-message').style.color = '#BD362F';
+            document.getElementById('title-input-message').innerHTML = 'Title cannot be blank.';
+        },
+        updateDescriptionError: function() {
+            document.getElementById('description-input-message').style.color = '#BD362F';
```

Comment:
You needn't use `document.getElementById` - jQuery is available. Likewise, don't use the `.style.color`; use jQuery's methods instead.

This line would be:

   $('description-input-message').css({color = '#BD362F'})

That said, you also shouldn't be setting the color manually :) Use Bootstrap's classes - for an error, you might use `text-danger`.

See how other form validations on the OSF are done, especially those which use an HTML element that is bound to a property of the viewmodel.